kolibri/core/assets/src/composables/useUserSyncStatus.js

```diff
@@ -64,11 +66,16 @@ export default function useUserSyncStatus() {
     }
   });
 
+  const userLoggedIn = computed(() => {
+    return !!store.state.core.session.user_id;
+  });
+
   return {
     queued,
     lastSynced,
     status,
     deviceStatus,
     deviceStatusSentiment,
+    userLoggedIn,
```

Comment:
I'm not sure where you are ultimately using this value, but we might be able to use the 
`isUserLoggedIn` [value](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/state/modules/session.js#L36) from `session.js` instead

Reply:
Note there is also a composable for this too, which would be better used wherever this is being used (and can also be used in here where needed): https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/composables/useUser.js